src/theme/ThemeInterface.ts

```diff
@@ -8,6 +8,8 @@ import { DeepPartial } from '../utility-types'
 export interface Palette {
   main: string
   contrastText: string
+  pairCellBg: string
```

Comment:
`pairCellBg` should be `evenCellBg`, right? 
I'm not a native English speaker so I may be wrong!

Reply:
@acro5piano yes, let's change that name for `even` and `uneven` 👍

Reply:
@acro5piano I will leave the names like this: `even` and `odd`, what do you think?